src/Http/Routing/test/UnitTests/Builder/RoutingEndpointConventionBuilderExtensionsTest.cs

```diff
@@ -9,6 +9,26 @@ namespace Microsoft.AspNetCore.Builder;
 
 public class RoutingEndpointConventionBuilderExtensionsTest
 {
+    [Fact]
+    public void RequireHost_HostNames()
```

Comment:
Seems random, what exactly is this test verifying? Maybe needs a more descriptive name

Reply:
This is a test that I moved over from [this file](https://github.com/dotnet/aspnetcore/blob/ac4bd5ef1660dbbd380de1e80c36468cc912770d/src/Http/Routing/test/UnitTests/RoutingEndpointConventionBuilderExtensionsTests.cs) that contained this singular test. It seemed weird to have this test on its own when it was testing the same surface area as these so I moved it over. I can take a stab at a better name now that it is in the larger file.